src/action/base/create_directory.rs

```diff
@@ -186,3 +187,59 @@ impl Action for CreateDirectory {
         Ok(())
     }
 }
+
+#[tokio::test]
```

Comment:
Should these be in a `mod test` guarded by `#[cfg(test)]`?

Reply:
Yes :sweat_smile: